lib/honeybadger/cli/test.rb

```diff
@@ -142,6 +142,7 @@ def verify; end
         end
         CONTROLLER
 
+        ::Rails.application.try(:reload_routes_unless_loaded)
```

Comment:
I discovered this by debugging the routes call below:

```
::Rails.application.routes.tap do |r|
  puts r.routes.inspect
  ...
end
```

When I called `r.routes`, the test worked. When I dug into the method, it called the `#reload_routes_unless_loaded` method.